### ISSUE TITLE
Need to skip tests that require DBus

### ISSUE THREAD

**Issue by irl** [Azure/azure-sdk-for-python#1389]
Build environments, such as Debian's, will not have DBus available.
```
======================================================================
ERROR: tests.test_mgmt_media (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: tests.test_mgmt_media
Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "tests/test_mgmt_media.py", line 13, in <module>
    from devtools_testutils import (
  File "devtools_testutils/__init__.py", line 1, in <module>
    from .mgmt_testcase import (AzureMgmtTestCase, AzureMgmtPreparer)
  File "devtools_testutils/mgmt_testcase.py", line 17, in <module>
    from . import mgmt_settings_fake as fake_settings
  File "devtools_testutils/mgmt_settings_fake.py", line 7, in <module>
    from azure.common.credentials import (
  File "azure/common/credentials.py", line 60, in <module>
    from msrestazure.azure_active_directory import (
  File "/usr/lib/python2.7/dist-packages/msrestazure/azure_active_directory.py", line 37, in <module>
    import keyring
  File "/usr/lib/python2.7/dist-packages/keyring/__init__.py", line 6, in <module>
    from .core import (set_keyring, get_keyring, set_password, get_password,
  File "/usr/lib/python2.7/dist-packages/keyring/core.py", line 148, in <module>
    init_backend()
  File "/usr/lib/python2.7/dist-packages/keyring/core.py", line 64, in init_backend
    keyrings = filter(limit, backend.get_all_keyring())
  File "/usr/lib/python2.7/dist-packages/keyring/util/__init__.py", line 20, in wrapper
    func.always_returns = func(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/keyring/backend.py", line 191, in get_all_keyring
    exceptions=TypeError))
  File "/usr/lib/python2.7/dist-packages/keyring/util/__init__.py", line 29, in suppress_exceptions
    for callable in callables:
  File "/usr/lib/python2.7/dist-packages/keyring/backend.py", line 183, in is_class_viable
    keyring_cls.priority
  File "/usr/lib/python2.7/dist-packages/keyring/util/properties.py", line 22, in __get__
    return self.fget.__get__(None, owner)()
  File "/usr/lib/python2.7/dist-packages/keyring/backends/SecretService.py", line 30, in priority
    bus = secretstorage.dbus_init()
  File "/usr/lib/python2.7/dist-packages/secretstorage/__init__.py", line 47, in dbus_init
    return dbus.SessionBus()
  File "/usr/lib/python2.7/dist-packages/dbus/_dbus.py", line 211, in __new__
    mainloop=mainloop)
  File "/usr/lib/python2.7/dist-packages/dbus/_dbus.py", line 100, in __new__
    bus = BusConnection.__new__(subclass, bus_type, mainloop=mainloop)
  File "/usr/lib/python2.7/dist-packages/dbus/bus.py", line 122, in __new__
    bus = cls._new_for_bus(address_or_type, mainloop=mainloop)
DBusException: org.freedesktop.DBus.Error.FileNotFound: Failed to connect to socket /run/user/1000/bus: No such file or directory
```

**Comment by lmazuel** [comment]
Hi @irl 
I released msrestazure 0.4.12 yesterday that should take care of this. Could you confirm?
Thanks!

**Comment by lmazuel** [comment]
Ho actually the `import keyring` itself is failling.... So msrestazure 0.4.12 will not be enough. I created an msrestazure issue:
https://github.com/Azure/msrestazure-for-python/issues/42

I'll try to fix that today with a 0.4.13

**Comment by irl** [comment]
Ok thanks. I have updated Debian unstable now to msrestazure 0.4.12, so that's what I was building with. I have a local package for the current git HEAD of azure-python-devtools also, which is the only difference between my build environment and Debian unstable, if you wanted to check other version numbers.

**Comment by lmazuel** [comment]
@irl I released "msrestazure" 0.4.13 that should fix your problem.

**Comment by irl** [comment]
That solves that one.
